packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -135,6 +161,31 @@ class ToolbarItem extends UIComponent<WithAsProp<ToolbarItemProps>, ToolbarItemS
       </ElementType>
     )
 
+    const submenu = menuOpen ? (
```

Comment:
what if the menu's items is an empty array?

Reply:
Then in renders empty submenu, consistent with `Menu`. Do you think it is incorrect?

Reply:
I would say if there are no menu items we don't need to render it. What the use case of rendering an empty menu?

Reply:
Agree, but I think it is consumer's responsibility. `menu` can be array or object or element and we cannot check all. So I think it is better to be consistently dump than to try to be smart but inconsistent.